### PR TITLE
reformat of src and test using Apache NetBeans default

### PR DESCRIPTION
**Pull request by ebarboni**
Replacement of #121 by only reformating and reconfiguring checkstyle to use default google, (sun is complaining about javadoc and finals)

using blame ignore to no pollute this no op changes

### CODE REVIEW

File: .git-blame-ignore-revs [hunk 1]
```diff
@@ -19,3 +19,6 @@
 
 #
 ee3ecabf930faebc2e8c8c455c1a03dd591303d7
+
+## Reformat src and test to Apache NetBeans default ALT-SHIFT-F
+364b5432961e566c4f9b9eadbae566c0ed93db96
```
Comment (mbien) [inline .git-blame-ignore-revs:24]: the commit hash will probably change on merge

File: pom.xml [hunk 2]
```diff
@@ -150,7 +150,7 @@
                     <artifactId>maven-checkstyle-plugin</artifactId>
                     <version>3.3.1</version>
                     <configuration>
-                        <configLocation>config/maven_checks.xml</configLocation>
+                        <configLocation>google_checks.xml</configLocation>
                         <headerLocation>config/maven-header.txt</headerLocation>
                     </configuration>
                     <dependencies>
```
Comment (mbien) [inline pom.xml:156]: question: what happens if someone breaks the rules? will the build fail? (please don't do that ;))

**Review by mbien — COMMENTED** [review]
sorry I tried to hold my promise and try to format it using the netbeans formatter (so that I learn a bit more about it) - but never had the time since there was always something else to do

File: .git-blame-ignore-revs (hunk 1 above)
Comment (ebarboni) [reply .git-blame-ignore-revs:24]: no I use the default nebeans but the config checkstyle should use to be "muted" is google

File: pom.xml (hunk 2 above)
Comment (ebarboni) [reply pom.xml:156]: no we had a lots of issue before but no fail.

Comment (ebarboni) [reply pom.xml:156]: ``<failsOnError> `` true would make it fail but it's default false

**Comment by ebarboni** [comment]
merging this ?

**Comment by mbien** [comment]
i think the `.git-blame-ignore-revs` trick won't work since the commit hash will change post integration, but other than that looks good i think.

**Comment by ebarboni** [comment]
ok will direct commit the ignore if not working

**Pull request merged**